src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Schemas/JSON/template.json

```diff
@@ -739,6 +739,10 @@
       "pattern": "^([0-9]+)$",
       "default": 0
     },
+    "preferDefaultName": {
+      "description": "Indicated which behaviour to use when no `--name` is specified during template creation.",
```

Comment:
```suggestion
      "description": "Indicated which behaviour to use when no `--name` is specified during template creation. If `false` it will use the fallback (output folder) name, if `true` it will use the template's `defaultName`.",
```